src/main/perl/FileWriter.pm

```diff
@@ -173,8 +172,8 @@ sub close
                                   keeps_state => 1);
         $cmd->execute();
         *$self->{LOG}->verbose ("Changes to ", *$self->{filename}, ":");
+        $diff = "" if (! defined($diff));
```

Comment:
Does this cause report to log an empty line or is it suppressed by the logging stack? Is an empty line the right way of indicating an empty diff or should we print something standardised such as "(no changes)" ?